Admin/AdminInterface.php

```diff
@@ -145,7 +143,7 @@ public function getBaseControllerName();
     public function generateObjectUrl($name, $object, array $parameters = array(), $absolute = false);
 
     /**
-     * Generates an url for the given parameters.
+     * Generates a url for the given parameters.
```

Comment:
That's not how it works. U starts with a consonant sound. In phonetic, it's spelt "ju", check it out ;)